OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignalPrefs.java

```diff
@@ -74,7 +74,7 @@
     public static final String PREFS_ONESIGNAL_USERSTATE_DEPENDVALYES_ = "ONESIGNAL_USERSTATE_DEPENDVALYES_";
     public static final String PREFS_ONESIGNAL_USERSTATE_SYNCVALYES_ = "ONESIGNAL_USERSTATE_SYNCVALYES_";
     public static final String PREFS_ONESIGNAL_ACCEPTED_NOTIFICATION_LAST = "ONESIGNAL_ACCEPTED_NOTIFICATION_LAST";
-    public static final String PREFS_ONESIGNAL_SUBSCRIPTION_LAST = "ONESIGNAL_SUBSCRIPTION_LAST";
+    public static final String PREFS_ONESIGNAL_PUSH_DISABLED = "PREFS_ONESIGNAL_PUSH_DISABLED";
```

Comment:
Must keep the string `"PREFS_ONESIGNAL_PUSH_DISABLED"` the same otherwise the value won't carry over when the app update from 3.x to 4.x.

I recommend we keep this the same for now since this is only internally used